felix/.gitignore

```diff
@@ -70,3 +70,6 @@ htmlcov/
 
 # produced as part of openstack packaging build
 debian/files
+
+#Env Configuration
+.env
```

Comment:
Minor: I think we usually have a next line at the end of the file.

Reply:
fixed